micronaut-camunda-bpm-feature/src/test/java/info/novatec/micronaut/camunda/bpm/feature/MicronautProcessEngineConfigurationCustomizer.java

```diff
@@ -0,0 +1,18 @@
+package info.novatec.micronaut.camunda.bpm.feature;
+
+import org.camunda.bpm.engine.ProcessEngineConfiguration;
+
+import javax.inject.Singleton;
+import java.util.function.Consumer;
+
+@Singleton
+public class MicronautProcessEngineConfigurationCustomizer implements Consumer<ProcessEngineConfiguration>  {
```

Comment:
I think defining an interface like `ProcessEngineConfigurationCustomizer` has some advantages over using the plain `Consumer` interface such as:

1. It adds semantics. The developer sees immediately what the interface is supposed to do.
2. If defined as `FunctionalInterface`, it can be still used with lambdas.

Reply:
I also think it's better approach. I've updated the code accordingly.